.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "third_party/prometheus-cpp"]
 	path = third_party/prometheus-cpp
 	url = https://github.com/jupp0r/prometheus-cpp.git
+[submodule "third_party/benchmark"]
```

Comment:
You should be able to remove that, unless you are adding Benchmark tests here.

Reply:
removed